tests/scripts/thread-cert/border_router/README.md

```diff
@@ -0,0 +1,25 @@
+# OpenThread Border Router Tests
+
+## Run Border Router (BR) tests locally
+
+BR tests run in isolated Docker network and containers, so new OTBR Docker image needs to be created before running the tests:
```

Comment:
LGTM - might want to say: 
BR tests run in isolated Docker network and containers, so a new OTBR Docker image needs to be created before running these tests:

Reply:
```suggestion
BR tests run in isolated Docker network and containers, so a new OTBR Docker image needs to be created before running these tests:
```